lib/xnu/reload.sh

```diff
@@ -1,7 +1,23 @@
 #!/bin/sh
-sudo kextunload /Library/Extensions/FridaKernelAgent.kext &>/dev/null
-sudo rm -rf /Library/Extensions/FridaKernelAgent.kext
-sudo cp -a ~/Library/Developer/Xcode/DerivedData/FridaKernelAgent-eaueaeplpgvooydjbumkfbqjrobj/Build/Products/Debug/FridaKernelAgent.kext /Library/Extensions/FridaKernelAgent.kext
-sudo chown -R root /Library/Extensions/FridaKernelAgent.kext
-sudo chgrp -R wheel /Library/Extensions/FridaKernelAgent.kext
-sudo kextload /Library/Extensions/FridaKernelAgent.kext
+
+KEXT_NAME=FridaKernelAgent.kext
+
+sudo kextunload "/Library/Extensions/$KEXT_NAME" &>/dev/null
+sudo rm -rf "/Library/Extensions/$KEXT_NAME"
+
+BUILD_DIR="$(mktemp -d)"
+
+cd "$(dirname "$(realpath "$0")")"
+xcodebuild build\
+  CONFIGURATION_BUILD_DIR="$BUILD_DIR"\
+  CODE_SIGN_IDENTITY="$CODE_SIGN_IDENTITY"\
+  &>/dev/null
```

Comment:
For consistency it would be good to put a space before each continuation-backslash.

Reply:
I forgot sh allows it :|